### ISSUE TITLE
Пустой раздел про планирование расширения в третьей главе

### ISSUE THREAD

**Issue by elricbk** [eucariot/SDSM#55]
Файл https://github.com/eucariot/SDSM/blob/master/3.-static_routing/01-planirovanie-rashireniya.md содержит только заголовок, а на linkmeup соответствующая часть есть. В принципе, это #40, но там как раз предполагалось отдельные issue заводить.

**Comment by eucariot** [comment]
Да, кому-то определённо этим нужно заняться. Я пока сместил приоритет на новый цикл.
